manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -162,30 +202,37 @@ LoadModule auth_mellon_module modules/mod_auth_mellon.so
 	return fmt.Sprintf(s, httpdAuthRemoteUserConf())
 }
 
-func httpdOIDCAuthConf(providerURL, applicationDomain string) string {
+func httpdOIDCAuthConf(providerURL, introspectionURL, applicationDomain string) string {
 	// If these are not provided, we should assume that the user provided a full config
 	// in a secret, so include the directory for that secret here
-	if providerURL == "" || applicationDomain == "" {
+	if providerURL == "" || introspectionURL == "" || applicationDomain == "" {
 		return "Include user-conf.d/*.conf"
 	}
 
 	s := `
 LoadModule auth_openidc_module modules/mod_auth_openidc.so
+ServerName https://%s
+LogLevel   warn
 
 OIDCProviderMetadataURL      %s
 OIDCClientID                 ${HTTPD_AUTH_OIDC_CLIENT_ID}
 OIDCClientSecret             ${HTTPD_AUTH_OIDC_CLIENT_SECRET}
-
 OIDCRedirectURI              "https://%s/oidc_login/redirect_uri"
+OIDCCryptoPassphrase         sp-secret
 OIDCOAuthRemoteUserClaim     username
 
-OIDCCryptoPassphrase         sp-secret
+OIDCOAuthClientID                  ${HTTPD_AUTH_OIDC_CLIENT_ID}
+OIDCOAuthClientSecret              ${HTTPD_AUTH_OIDC_CLIENT_SECRET}
+OIDCOAuthIntrospectionEndpoint     %s
```

Comment:
Typically the introspection endpoint is derived from the provider url, so there would be no reason to specify it normally.  That is, I would expect that the value would be left blank, and then only if specified would we have this line.

Reply:
I think that's essentially what @abellotti is saying in https://github.com/ManageIQ/manageiq-appliance/pull/282#issuecomment-625236558 so I was going to wait to see what direction they go.

If we want to make this optional in the non-entire-conf case I can do what he's suggesting I think.

Reply:
Yeah, I think I'd like a sensible default if not specified based on what @abellotti wrote there, as opposed to requiring it to be specified.